src/transport/noise.rs

```diff
@@ -99,4 +99,8 @@ impl Transport for NoiseTransport {
             .with_context(|| "Failed to do noise handshake")?;
         return Ok(conn);
     }
+
+    async fn close(&self, a: Self::Acceptor) {
+        drop(a); // gratuitous drop
```

Comment:
Is this line necessary? What do you think about an empty block?

Reply:
To be honest, I have no clue.

I am not very familiar with implementing `Drop` and I am not sure whether that drop happens implicitly anyways because acceptor is consumed and goes out of scope when `close` exits.

Reply:
`a` is passed in and when the function exits, `a` will be out of scope and dropped. So you can use an empty block here

Reply:
Will do